Automattic/newspack-ads
# Sticky ad widget w/ non-AMP ads

Makes use of the experimental AMP Plus (https://github.com/Automattic/newspack-plugin/pull/990) to serve a sticky sidebar ad.

## How to test

1. Switch `newspack-plugin` to `add/amp-plus-with-error-sanitisation` branch (https://github.com/Automattic/newspack-plugin/pull/990)
2. Set up an ad in the sidebar, as a widget
3. Observe the site loads as usual in AMP Standard & Transitional modes
4. Add `define( 'NEWSPACK_AMP_PLUS_ENABLED' );` environment variable
5. Observe the widget having a new setting now: "Stick to top" checkbox - turn it on for the 
7. Visit the site in AMP Standard mode
8. Observe the widget ad being stuck to top of the viewport after scrolling down ✨

---

Supersedes #81
